### PR TITLE
Add option to swap default Alt+scroll zooming behavior in 2D editor

### PR DESCRIPTION
**Pull request by alula**

### CODE REVIEW

File: doc/classes/EditorSettings.xml [hunk 1]
```diff
@@ -237,6 +237,9 @@
 		<member name="editors/2d/smart_snapping_line_color" type="Color" setter="" getter="">
 			The color to use when drawing smart snapping lines in the 2D editor. The smart snapping lines will automatically display when moving 2D nodes if smart snapping is enabled in the Snapping Options menu at the top of the 2D editor viewport.
 		</member>
+		<member name="editors/2d/use_integer_zoom_by_default" type="bool" setter="" getter="">
+			If [code]true[/code], the 2D editor will snap to integer zoom values while not holding the [kbd]Alt[/kbd] key and powers of two while holding it. Otherwise the behavior is swapped.
```
Comment (Calinou) [inline doc/classes/EditorSettings.xml:241]:
```suggestion
			If [code]true[/code], the 2D editor will snap to integer zoom values while not holding the [kbd]Alt[/kbd] key and powers of two while holding it. If [code]false[/code], this behavior is swapped.
```

**Review by Calinou — APPROVED** [review]
Tested locally (on top of `master` 329652b6feac2114c4e30062f18ece36d4e3addc), it works. Code and documentation look good to me.

**Comment by akien-mga** [comment]
Looks good! Could you squash the commits? See [PR workflow](https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#the-interactive-rebase) for instructions.

For the record the CI error is not related to this issue, a new build attempt should hopefully solve it.

**Comment by alula** [comment]
done

**Comment by YuriSizov** [comment]
Thanks!

**Pull request merged**